Sources/ArcGISToolkit/Components/Authentication/CertificatePickerViewModifier.swift

```diff
@@ -74,15 +84,16 @@ import ArcGIS
         Task.detached {
```

Comment:
Is there any way the enclosing method could be made `async` so as not to create a top-level task?

Reply:
The function is being called from the view modifier. If we make this function `async` then will have to move the `Task` in upper level too. I feel, it good to keep it here.

Reply:
Probably outside the scope of this PR, but perhaps the view modifier could be refactored to use a `task` modifier.

Reply:
Logged an [issue](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/249)